examples/blueprint-construct/index.ts

```diff
@@ -119,7 +119,9 @@ export default class BlueprintConstruct {
                 consolidation: { enabled: true },
                 ttlSecondsUntilExpired: 360,
```

Comment:
Same as above, I suggest this be set to `2592000`.

Reply:
addressed.